### PR TITLE
Gramatical correction in /roomauth and /authlist

### PR DESCRIPTION
**Pull request by panpawn**
Before, /roomauth (and /authlist) would display that there would be plural of a rank even if there was only one user of that rank.  (Example: http://i.imgur.com/zxfJWSV.png) Now, roomauth and /authlist will look for if there is more than one of each rank and to mark if it is in fact plural or not accordingly.

(Also, slight grammatical change in the return message of /authlist if the server has no global authority.)

### CODE REVIEW

**Comment by TheImmortal** [comment]
Plural looks much better imo.

**Comment by panpawn** [comment]
And it will be plural if there are in fact more than one of said users of the rank it's displaying.

**Comment by TheImmortal** [comment]
Yeah but even if there's one, "Driver" and "Moderator" looks dumb imo.

File: commands.js [hunk 1]
```diff
@@ -49,10 +49,10 @@ var commands = exports.commands = {
 		Object.keys(rankLists).sort(function (a, b) {
 			return (Config.groups[b] || {rank: 0}).rank - (Config.groups[a] || {rank: 0}).rank;
 		}).forEach(function (r) {
-			buffer.push((Config.groups[r] ? Config.groups[r].name + "s (" + r + ")" : r) + ":\n" + rankLists[r].sortBy(toId).join(", "));
+			buffer.push((Config.groups[r] ? Config.groups[r].name + (rankLists[r].sort().length > 1 ? "s" : "") + " (" + r + ")" : r) + ":\n" + rankLists[r].sortBy(toId).join(", "));
 		});
 
-		if (!buffer.length) buffer = "This server has no auth.";
+		if (!buffer.length) buffer = "This server has no global authority.";
```
Comment (Zarel) [inline commands.js]: This line is good, but I agree with @TheImmortal that I don't like the other changes.

**Comment by Zarel** [comment]
I'm like 60% sure Sugar.js has a pluralize function.

More importantly, I agree with @TheImmortal - I prefer "Room owners:" even if there's only one.

**Comment by panpawn** [comment]
Alright, and should I also be making the return message for if a room doesn't have room authority similar to /authlist?

**Comment by Zarel** [comment]
If you'd like.

**Pull request closed without merging**